src/explore-education-statistics-frontend/src/modules/methodologies/MethodologyIndexPage.tsx

```diff
@@ -111,7 +113,12 @@ export const getServerSideProps: GetServerSideProps<Props> = async () => {
 export default MethodologyIndexPage;
 
 function getMethodologiesForTopics(topics: MethodologyTopic[]) {
-  return topics.flatMap(topic =>
+  const methodologies = topics.flatMap(topic =>
```

Comment:
Personally, I would have leaned towards de-duping in the backend by removing topics as we're in the area anyway.

Reply:
The backend isn't duplicating, really: it's just it provides the categories theme -> topic -> publication -> methodology - so if a methodology is in more than one topic or methodology under one theme, it will appear multiple times in the flattened results. So if we were to update the backend, we'd have to change the categories provided in the response to theme -> methodology - which is all the frontend requires, but I thought wasn't worth the effort.